src/test/java/features/Dropdown Tests.feature

```diff
@@ -15,7 +15,12 @@ Feature: Dropdown Tests
 	
   @22
   Scenario: 22 Prime NG Dropdown
-    Given I am on the Prime NG Dropdown Page
+    Given I am on the Prime NG Home Page
+    When I click the Components Button
+    Then I am redirected to the Prime NG Setup Page
```

Comment:
If we have a link directly to the Prime NG Setup page, then we should be going directly there and cut out extraneous steps. We are specifically testing our interaction with the PrimeNG dropdown implementation. We are not testing navigation there.

If we cannot get to the setup page directly with the given URL, then we should delete the URL from that page object.

Reply:
This is because of that navigation issue where if we go directly it redirects and breaks, We can remove the validation but we do need to navigate from home -> setup -> dropdown